src/utilities.h

```diff
@@ -555,4 +555,30 @@ static inline std::string trimCopy(std::string s)
     return s;
 }
 
+/**
+ * @brief Collect all existing id attributes within the given model.
+ *
+ * @param model The @c ModelPtr to interrogate.
+ * and their child elements.
```

Comment:
(forgetting my doxygen) but shouldn't this comment be part of the brief doxy string? or is that `.` the line above misplaced?

Reply:
The . is out of line, thanks.

Reply:
37a7d869